Mathlib/FieldTheory/Galois/Infinite.lean

```diff
@@ -0,0 +1,271 @@
+        use 1
+        simp only [SetLike.mem_coe, smul_eq_mul, mul_one, and_true, Subgroup.one_mem]
+
+lemma fixedField_fixingSubgroup (L : IntermediateField k K) [IsGalois k K] :
+    IntermediateField.fixedField L.fixingSubgroup = L := by
+  apply le_antisymm
+  · intro x hx
+    rw [IntermediateField.mem_fixedField_iff] at hx
+    have mem : x ∈ (adjoin L {x}).1 := subset_adjoin _ _ rfl
+    have : IntermediateField.fixedField (⊤ : Subgroup ((adjoin L {x}) ≃ₐ[L] (adjoin L {x}))) = ⊥ :=
+      (IsGalois.tfae.out 0 1).mp (by infer_instance)
+    have : ⟨x, mem⟩ ∈ (⊥ : IntermediateField L (adjoin L {x})) := by
+      rw [← this, IntermediateField.mem_fixedField_iff]
+      intro f _
+      rcases restrictNormalHom_surjective K f with ⟨σ,hσ⟩
+      apply Subtype.val_injective
+      rw [← hσ, restrictNormalHom_apply (adjoin L {x}).1 σ ⟨x, mem⟩]
+      have := hx ((IntermediateField.fixingSubgroupEquiv L).symm σ)
+      simpa only [SetLike.coe_mem, true_implies]
+    rcases IntermediateField.mem_bot.mp this with ⟨y, hy⟩
+    obtain ⟨rfl⟩ : y = x := congrArg Subtype.val hy
+    exact y.2
+  · exact (IntermediateField.le_iff_le L.fixingSubgroup L).mpr fun ⦃x⦄ a ↦ a
+
+lemma fixedField_bot [IsGalois k K] :
+    IntermediateField.fixedField (⊤ : Subgroup (K ≃ₐ[k] K)) = ⊥ := by
+  rw [← IntermediateField.fixingSubgroup_bot, fixedField_fixingSubgroup]
+
+lemma restrict_fixedField (H : Subgroup (K ≃ₐ[k] K)) (L : IntermediateField k K) [IsGalois k L] :
+    (IntermediateField.fixedField H) ⊓ L = IntermediateField.lift (IntermediateField.fixedField
+    (Subgroup.map (restrictNormalHom (F := k) (K₁ := K) L) H)) := by
+  apply SetLike.ext'
+  ext x
+  refine ⟨fun h ↦ ?_, fun h ↦ ?_⟩
+  · have xL := h.out.2
+    apply (IntermediateField.mem_lift (⟨x, xL⟩ : L)).mpr
+    rw [IntermediateField.mem_fixedField_iff]
+    simp only [Subgroup.mem_map, forall_exists_index, and_imp, forall_apply_eq_imp_iff₂]
+    intro σ hσ
+    apply Subtype.val_injective
+    dsimp
+    nth_rw 2 [← (h.out.1 ⟨σ, hσ⟩)]
+    exact AlgEquiv.restrictNormal_commutes σ L ⟨x, xL⟩
+  · have xL := IntermediateField.lift_le _ h
+    apply (IntermediateField.mem_lift (⟨x,xL⟩ : L)).mp at h
+    simp only [IntermediateField.mem_fixedField_iff, Subgroup.mem_map, forall_exists_index, and_imp,
+      forall_apply_eq_imp_iff₂] at h
+    simp only [IntermediateField.coe_inf, Set.mem_inter_iff, SetLike.mem_coe,
+      IntermediateField.mem_fixedField_iff, xL, and_true]
+    intro σ hσ
+    have : ((restrictNormalHom L σ) ⟨x, xL⟩).1 = x := by rw [h σ hσ]
+    nth_rw 2 [← this]
+    exact (AlgEquiv.restrictNormal_commutes σ L ⟨x, xL⟩).symm
+
+lemma fixingSubgroup_fixedField (H : ClosedSubgroup (K ≃ₐ[k] K)) [IsGalois k K] :
+    (IntermediateField.fixedField H).fixingSubgroup = H.1 := by
+  apply le_antisymm _ ((IntermediateField.le_iff_le H.toSubgroup
+    (IntermediateField.fixedField H.toSubgroup)).mp fun ⦃x⦄ a ↦ a)
```

Comment:
```suggestion
    (IntermediateField.fixedField H.toSubgroup)).mp (by rfl)
```
Please take care of all other occurrencies. What you're doing is saying that something is `≤` than itself, and this is `rfl`.

Reply:
You could probably even use term mode `le_rfl`.

Reply:
found one relevant that can be modified too, thanks for the suggestion